docs/source/pythonapi/examples/tally-arithmetic.ipynb

```diff
@@ -1002,23 +1019,24 @@
        "  <tbody>\n",
        "    <tr>\n",
        "      <th>0</th>\n",
-       "      <td>(0.0e+00 - 6.2e-01)</td>\n",
-       "      <td>10000</td>\n",
-       "      <td>total</td>\n",
-       "      <td>(nu-fission / absorption)</td>\n",
-       "      <td>1.237053</td>\n",
-       "      <td>0.011765</td>\n",
+       "      <td> 0</td>\n",
+       "      <td> 0.000001</td>\n",
+       "      <td> 10000</td>\n",
+       "      <td> total</td>\n",
+       "      <td> (nu-fission / absorption)</td>\n",
+       "      <td> 1.663616</td>\n",
```

Comment:
This example accidentally used 625 keV rather than 0.625 eV as the thermal cut-off.  I fixed it to 0.625 eV which caused these changes